templates/package-json/package.json

```diff
@@ -5,10 +5,10 @@
   "license": "UNLICENSED",
   "scripts": {
     "analyze": "node scripts/create-app-components && webpack --mode=production --env.analyzer",
-    "analyze:static": "node scripts/create-app-components && webpack --progress --mode=production --env.static --env.analyzer",
+    "analyze:static": "node scripts/create-static-site-files && webpack --progress --mode=production --env.static --env.analyzer",
```

Comment:
What about `scripts/create-app-components`?

Reply:
Not needed when building a static site. Mounting of the whole page is done in `static-client.js`.